api/openapi.yaml

```diff
@@ -2407,6 +2781,160 @@ components:
+          type: string
+        stateReason:
+          description: Machine-readable, UpperCamelCase text indicating the reason for the condition's last transition.
+          type: string
+          enum:
+            - Aborted
+            - TimedOut
+            - OneOrMoreAssetFailedToEstimate
+            - DiscoveryFailed
+            - Unexpected
+            - NothingToEstimate
+            - Success
+        startTime:
+          type: string
+          format: date-time
+        endTime:
+          type: string
+          format: date-time
+        estimation:
+          $ref: '#/components/schemas/Estimation'
+        scanTemplate:
+          $ref: '#/components/schemas/ScanTemplate'
+        summary:
+          $ref: '#/components/schemas/ScanEstimationSummary'
+        revision:
+          type: integer
+
+    ScanEstimationExists:
+      type: object
+      properties:
+        message:
+          description: Describes which unique constraint combination causes the conflict.
+          type: string
+          readOnly: true
+        scanEstimation:
+          $ref: '#/components/schemas/ScanEstimation'
+
+    Estimation:
+      type: object
+      properties:
+        time:
+          description: Total time the scan will take (seconds)
+          type: integer
+        size:
+          description: Total size of the scan (GB)
+          type: integer
+        cost:
+          description: Total cost of the scan ($)
+          type: number
+          format: float
+        recipe:
+          type: array
+          items:
+            $ref: '#/components/schemas/RecipeComponent'
+
+    RecipeComponent:
+      type: object
+      properties:
+        operation:
+          type: string
+        cost:
+          type: number
+          format: float
+
+    AssetScanEstimations:
+      type: object
+      properties:
+        count:
+          description: Total AssetScanEstimations count according to the given filters
+          type: integer
+        items:
+          description: List of AssetScanEstimations according to the given filters
+          type: array
+          items:
+            $ref: '#/components/schemas/AssetScanEstimation'
+
+    ScanEstimationRelationship:
+      type: object
+      properties:
+        id:
+          type: string
+
+    AssetScanEstimation:
+      type: object
+      properties:
+        id:
+          type: string
+        asset:
+          $ref: '#/components/schemas/Asset'
+        # TODO (erezf) replace with ScanEstimationRelationship object (?)
+        scanEstimation:
+          $ref: '#/components/schemas/ScanEstimationRelationship'
+        state:
+          $ref: '#/components/schemas/AssetScanEstimationState'
+        estimation:
+          $ref: '#/components/schemas/Estimation'
+        assetScanTemplate:
+          $ref: '#/components/schemas/AssetScanTemplate'
+        revision:
+          type: integer
```

Comment:
We should add StartTime/EndTime here so that we can use them for timeout purposes.